SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c

```diff
@@ -2615,6 +2615,17 @@ OnExitBootServicesFailed (
 {
   EFI_STATUS  Status;
 
+  //
+  // Measure invocation of ExitBootServices,
+  //
+  Status = TcgMeasureAction (

```

Comment:
Did the parameters need to be on separate lines per Uncrustify? It seems like they would be fine on a single line in this case.

Reply:
Not a big deal either way, I was simply curious.

Reply:
Sure, I'll leave it as is to match the existing style.